heron/api/src/java/com/twitter/heron/streamlet/Config.java

```diff
@@ -107,6 +108,19 @@ public Builder setUserConfig(String key, Object value) {
       return this;
     }
 
+    /**
+     * Sets the topology to use the Kryo serializer for serializing
+     * streamlet elements
+     */
+    public Builder useKryoSerializer() {
+      try {
+        config.setSerializationClassName(new KryoSerializer().getClass().getName());
+      } catch (NoClassDefFoundError e) {
+        throw new RuntimeException("Please link with kryo");
```

Comment:
Suggest to be more explicit in error message, like "Linking with kryo is needed because useKryoSerializer is used".

Reply:
Changed